### DISCUSSION TITLE
How the value of P_pls_rbc_factor and P_pls_rbc_factor are determined?

### DISCUSSION THREAD

**Discussion by jessechao830330** [Open-Systems-Pharmacology/Forum#863] [Questions & Problems] [answered]
Hello team,

I found the example of the study protocol of diclofenac on the website: [https://github.com/Open-Systems-Pharmacology/MoBi/tree/develop/examples](url)  However, I have a few questions about some of the parameters in the model. I found parameters called "P_rbc_pls_factor" and "P_pls_rbc_factor" in Spatial Structures-Study Protocol 50 mg Diclofenac IR Tablet Single Dose/Parameters/Molecule Properties/Parameters. In its description, it only says that "Factor modifying the permeability of acids and bases depending on their degree of dissociation". Thus, I am wondering that if these parameters came from experimental data fitting or there are some equations to determine these parameters? Could you please explain how did you come up with the value of these parameters? 

Thank you very much! 

![Screenshot 2021-01-18 at 11 38 22](https://user-images.githubusercontent.com/66793768/104904656-b0fdac00-5981-11eb-87bd-abab8b89af5c.png)

**Comment by Vincent-AC** [comment]
Hello,
I am looking into a mobi model with the PK-Sim Standard calculation method for cellular permeability and have the same question, I cannot find how these P_bc_pls_factor and P_pls_bc_factor are calculated.
Thanks in adavance,

**Comment by msevestre** [comment, marked as answer]
This is called a "Black Box" Formula because definition will be retrieved when you build your simulation. This is unfortunately not very intuitive to find the formula.

But there is a way
1: Create your simulation and check for the instance of the parameter. You will see the formula
2: Open the AllCalculationMethods.pkml file and look for this formula
https://github.com/Open-Systems-Pharmacology/MoBi/blob/develop/src/Data/AllCalculationMethods.pkml

**Reply by Vincent-AC** [reply]
Hi, thanks !
I did look in the .pkml file but did not find the formula. I do realize now that it is because it is a constant value (1) hence why I did not find the formula.
Thanks a lot !
